doc/install/configure_ide.rst

```diff
@@ -0,0 +1,48 @@
+.. _configure-ide:
+
+IDE Configuration Guide
+=======================
+
+.. include:: ../links.inc
+.. highlight:: console
+
+Thanks for taking the time to contribute! This guide steps through various IDE settings that are helpful when
+contributing to MNE-Python.
+
+Some of the basic settings include enabling linting, setting line margins, introspection, and docstring type:
+
+* Linting - is the automated checking of your source code for programmatic and stylistic errors. Most of the time basic linting is already enabled but let's double check.
+
+* Line margins - we have code style rules that forbid lines longer than 79 characters, so having a ruler at 79 characters helps coders adhere to that rule.
+
+* Introspection - this helps removes trailing whitespaces that could be problematic when compiling or building
+
+* Docstring - we use NumPy because it supports a combination of reStructured and GoogleDocstrings and supported by Sphinx
```

Comment:
```suggestion
* Line margins - we have code style rules that forbid lines longer than 79 characters, so having a ruler at 79 characters helps coders adhere to that rule.

* Introspection - this helps removes trailing whitespaces that could be problematic when compiling or building

* Docstring - we use NumPy because it supports a combination of reStructured and GoogleDocstrings and is supported by Sphinx
```
I think this bullet point also could have a bit more information.

Reply:
(the last one on "Docstring" that is - just seeing now GitHub shows the previous ones in the view here too 🙃 )